app/src/main/res/values/strings.xml

```diff
@@ -1042,10 +1042,10 @@
     <string name="share_copy_link_to_clipboard">Copy to clipboard</string>
     <!-- Toast shown after copying link to clipboard -->
     <string name="toast_copy_link_to_clipboard">Copied to clipboard</string>
-    <!-- An option from the three dot menu to into sync -->
-    <string name="sync_menu_sign_in">Sign in to sync</string>
```

Comment:
`<string name="sync_menu_sign_in" moz:removedIn="107" tools:ignore="UnusedResources">Sign in to sync</string>`

Unfortunately, we can't simply remove a string. Please see https://github.com/mozilla-mobile/fenix/wiki/Removing-strings#marking-an-unused-string-to-be-removed